### PR TITLE
Fix null check errors on compiling

### PR DESCRIPTION
**Pull request by wshands**
Add 'null' datatype to variables so errors are not generated when using the --strictnullcheck compiler option

dockstore/dockstore#1793 null checks

### CODE REVIEW

File: src/app/aliases/aliases.component.ts [hunk 1]
```diff
@@ -32,20 +32,26 @@ export class AliasesComponent extends Base implements OnInit {
...
+    /*
+    let myType: string | null;
+    myType = this.route.snapshot.paramMap.get('type') ;
+    // Avoid assigning null to this.type
+    this.type = (myType) ? myType : '';
+     */
+    this.type = this.route.snapshot.paramMap.get('type') ;
     this.alias = this.route.snapshot.paramMap.get('alias');
-    this.validType = this.types.includes(this.type);
+    this.validType = (this.type) ? this.types.includes(this.type) : false;
```
Comment (coverbeck) [inline src/app/aliases/aliases.component.ts]: Is this necessary? Doesn't `this.types.includes(null)` return just `false`, or does the compiler option flag this? (I don't know)

File: src/app/aliases/aliases.component.ts [hunk 2]
```diff
@@ -32,20 +32,26 @@ export class AliasesComponent extends Base implements OnInit {
   }
 
   ngOnInit() {
-    this.type = this.route.snapshot.paramMap.get('type');
+    /*
```
Comment (coverbeck) [inline src/app/aliases/aliases.component.ts]: I think you can remove this block.

File: src/app/aliases/aliases.component.ts (hunk 1 above)
Comment (garyluu) [reply src/app/aliases/aliases.component.ts]: paraphrased: "the function 'include' does not exist on types, your website will now self-destruct"

File: src/app/aliases/aliases.component.ts (hunk 2 above)
Comment (wshands) [reply src/app/aliases/aliases.component.ts]: block removed

**Pull request merged**